src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -27,6 +27,17 @@
 
 #include <mpi.h>
```

Comment:
Is there a way to do this at the CMake level instead? Or can we generate some kind of file like this that gets used at configure time to check for support?

Reply:
Yes, we should be able to use `CHECK_CXX_SOURCE_COMPILE` here. This is what I was proposing in https://github.com/arborx/ArborX/pull/177#issuecomment-547615802.